extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -75,6 +83,8 @@ export class User implements IUserContext {
                 const params = cookie.split(User.cookieSeparator);
                 if (params.length > 0) {
                     _self.id = params[0];
+                    // we already have a cookie
+                    _self.isUserCookieSet = !!_self.id;
```

Comment:
If we successfully read a value from the cookie then we assume that the cookie was set.